locations/jclouds/src/main/java/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -836,6 +910,52 @@ protected JcloudsSshMachineLocation obtainOnce(ConfigBag setup) throws NoMachine
         }
     }
 
+    private void waitForWinRmAvailable(final NodeMetadata node, final LoginCredentials expectedCredentials, ConfigBag setup) {
+        // TODO: Reduce / remove duplication between this and waitForReachable
```

Comment:
Agree feels like a fair amount of duplication between this and `waitForReachable`. Worth considering if can have a common impl, e.g. passing in the checker, and the config key for wait time.